Upgrading-Cacti-Under-Windows.md

```diff
@@ -22,13 +25,24 @@ Download latest cacti-xxx.zip
 
 4. Windows Update
 
-5. MySQL Installer - update Catalog, then upgrade MySQL Server
+5. MySQL
+
+   * Open `MySQL Installer - Community`, update Catalog,
```

Comment:
Since we have separated out the steps, may as well fully separate them using bullet points.